server/Makefile

```diff
@@ -58,11 +58,6 @@ restart:
 	@echo "Restarting beanstalk..."
 	aws elasticbeanstalk restart-app-server --environment-name Instant-prod-env
 
-dev-with-prod-db:
```

Comment:
Removing this because you could mess up the sketches in prod if you run with the prod db locally.